lib/zoho_hub/base_record.rb

```diff
@@ -42,6 +42,17 @@ def find(id)
       def where(params)
         path = File.join(request_path, 'search')
 
+        if params.one?
```

Comment:
I would prefer to not use a Rails only method here.

Reply:
Good point !
:white_check_mark: